### DISCUSSION TITLE
wasm预览效果不正确

### DISCUSSION THREAD

**Discussion by cgeffect** [Tencent/libpag#2042] [Q&A] [answered]
[test.pag.zip](https://github.com/Tencent/libpag/files/13775084/test.pag.zip)

使用3.x的版本, commit 62c9f0c5818e51af681126113bdc92942bceaf18的版本, 使用这个PAG在linux上渲染效果正确, 但是打包wasm给前端预览是不正确的, 都是从62c9f0c5818e51af681126113bdc92942bceaf18这个commit下的代码. 

错误的效果: wasm预览效果, 出现了白屏
https://github.com/Tencent/libpag/assets/15692322/90fed1e8-d3a1-431a-b129-e9d1807dae79

正确的效果
https://github.com/Tencent/libpag/assets/15692322/f5138ea1-a41b-43cb-bf81-a3b6b01e3a6c

把代码切到cbc4be7f6878913a3594785c83c0eab78257affd这个commit发现预览正确, linux渲染也正确.
请问下在62c9f0c5818e51af681126113bdc92942bceaf18这个commit确实存在使用这个PAG预览不正确的问题吗? 

我需要更新最新的版本吗?

**Comment by domchen** [comment, marked as answer]
建议始终用最新的稳定版
